modules/microshift-ingress-controller-config.adoc

```diff
@@ -91,7 +116,108 @@ ingress:
+|The `ingress` section of the {microshift-short} `config.yaml` file defines the configurable parameters for the implementation of the {OCP} `IngressController` API. All of the following parameters in this table are subsections in the `ingress` section of the {microshift-short} `config.yaml`.
+
+|`accessLogging`
+|This `ingress` subsection describes how client requests are logged. If the `status` field is empty, access logging is disabled. When the status field is set to `Enabled`, access requests are logged as configured with the `accessLogging` parameters and the `accessLogging.destination.type` is automatically set to `Container`.
+
+* When enabled, access logging is part of the `openshift-router` logs. Logs from this pod are captured in the sos report procedure for {microshift-short}.
+//Q: how come `status` is not first in this list? is it typical for this to be way at the bottom of the config section? it seems like the "on switch")
+
+|`accessLogging.destination`
+|A destination for logs. The destination for logs can be a local sidecar container or remote. Default value is null.
+
+|`accessLogging.destination.type`
+|The type of destination for logs. Valid values are `Container` or `Syslog`.
+
+* Setting this value to `Container` specifies that logs should go to a sidecar container. When the destination type is set to `Container`, a container is called `access-log` and is automatically created. Using container logs means that logs might be dropped if the rate of logs exceeds the container runtime capacity or the custom logging solution capacity. You must have a custom logging solution that reads logs from this sidecar.
+
+* Setting this value to `Syslog` specifies that logs are sent to a Syslog endpoint. You must configure a custom Syslog instance and specify an endpoint that can receive Syslog messages. You must have a custom Syslog instance.
+//Q: should we supply instructions or links for these actions?
+
+|`accessLogging.destination.container`
+|Describes parameters for the `Container` logging destination type. You must configure a custom logging solution that reads logs from this sidecar.
+
+|`accessLogging.destination.container.maxLength`
+|Optional configuration. The default value is `1024` bytes. Message length must be at least `480` and not greater than `8192` bytes.
+
+|`accessLogging.destination.syslog`
+|Describes parameters for the `Syslog` logging destination type. You must configure a custom Syslog instance with an endpoint that can receive Syslog messages.
+
+|`accessLogging.destination.syslog.address`
+|Required configuration when the `Syslog` destination type is set. Valid value is the IP address of the syslog endpoint that receives log messages.
+//Q: any conformance for values that we should know about?
```

Comment:
Not really, this is just a plain ip address. If OCP specifies something then we should copy it, as the behavior is the same.